### PR TITLE
Added check-solana-price menu bar command

### PR DESCRIPTION
**Pull request by stevedylandev**
## Description

Added one menu bar command that displays the price of SOL in the menu bar. 

## Screencast

https://www.loom.com/share/b9069b539a794f04bb84494ff2bb7413

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by peduarte** [comment]
Excellent stuff, thank you! Love the Loom video as well. Nice touch :) 

Just one comment about the Solana SVG Icon: I see you went for the black version. The Menu Bar color is based on the wallpaper. MacOS will determine to use black/white based on the contrast ratio. 

But one way to change the color of the Icon dynamically is using `tint`:

```
icon={{
  source: "solana.svg",
  tintColor: {
    light: "black",
    dark: "white",
  },
}}
```
I've added this as a suggestion :) 

I also left a suggestion about the icon!

Thanks!

File: extensions/solana-explorer/src/check-solana-price.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      }
+    })();
+  }, []);
+  return state;
+};
+
+export default function Command() {
+  const { price, isLoading } = fetchPrice();
+
+  return <MenuBarExtra icon={{ source: "solana.svg" }} tooltip="gm" title={price} isLoading={isLoading}></MenuBarExtra>;
```
Comment (peduarte) [inline extensions/solana-explorer/src/check-solana-price.tsx:33]:
```suggestion
  return (
    <MenuBarExtra
      icon={{
        source: "solana.svg",
        tintColor: {
          light: "black",
          dark: "white",
        },
      }}
      tooltip="gm"
      title={price}
      isLoading={isLoading}
    ></MenuBarExtra>
  );
```

File: extensions/solana-explorer/src/check-solana-price.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    price: "",
+    isLoading: true,
+  });
+  useEffect(() => {
+    (async () => {
+      try {
+        const res = await axios.get("https://price.jup.ag/v3/price?ids=SOL");
+        const data = res.data;
+        const priceData = Math.round(data.data.SOL.price * 100) / 100;
+        console.log(priceData);
```
Comment (peduarte) [inline extensions/solana-explorer/src/check-solana-price.tsx:16]: Can we remove this log? Thanks!

**Comment by stevedsimkins** [comment]
Awesome thanks!! Can absolutely make those changes :)

File: extensions/solana-explorer/src/check-solana-price.tsx (hunk 2 above)
Comment (stevedsimkins) [reply extensions/solana-explorer/src/check-solana-price.tsx]: done!

File: extensions/solana-explorer/src/check-solana-price.tsx (hunk 1 above)
Comment (stevedsimkins) [reply extensions/solana-explorer/src/check-solana-price.tsx]: Resolved with [964d271](https://github.com/raycast/extensions/pull/4115/commits/964d2716ec6f1ff7841262ebcc1c22fe3c3e6ebb)

**Comment by stevedsimkins** [comment]
@peduarte Alright should be good to go! 👍🏻

**Review by peduarte — APPROVED** [review]

**Comment by peduarte** [comment]
Very nice. Thanks!

**Comment by peduarte** [comment]
LGTM. @TrGiLong You may want to have a look, since you're the author.

**Comment by stevedsimkins** [comment]
Hey @peduarte! If we don't hear back from @TrGiLong should I start building a separate extension? I know we may want to avoid that in general so I can try to make contact outside of Github

**Comment by stevedsimkins** [comment]
Hey all! Going to close this PR since we are not hearing from the author of the extension. I have other extensions I would like to submit and this is blocking that at the moment. Will revisit or build a separate extension in the future :)

**Comment by peduarte** [comment]
@stevedsimkins Hey, sorry about this. Extension Authors normally want to be involved in future updates of their extension. This is the first time that it's taken so long to hear back.

I'm going to raise this internally to see how we can handle this better in the future.

**Comment by stevedsimkins** [comment]
@peduarte No worries! I was trying to avoid making a new extension by just adding the feature to a similar/existing extension as outline in your docs. I'll likely revisit this and see if I can add more to it and make it a more feature full stand alone extension :)

**Pull request closed without merging**